### DISCUSSION TITLE
Swiper support for astro.js

### DISCUSSION THREAD

**Discussion by ibrahimaftab** [nolimits4web/swiper#7735] [Q&A]
Can you please add a support for astro.js usage?
There are some issues when using react and react swiper with astro.js when I m using coverflow slider it does not add proper css  on server side rendering however on client side it works great but its not a proper user experience

**Comment by pascal-brand38** [comment]
Hello,

You can have a look at [astro-swiper](https://github.com/pascal-brand38/astro-swiper).

There is a [coverflow demo on the doc site](https://pascal-brand38.github.io/astro-dev/packages/astro-swiper/#DemoEffectCoverFlow), and one on [codesandbox](https://codesandbox.io/p/sandbox/github/pascal-brand38/astro-codesandbox/tree/astro-swiper-coverflow?file=/src/components/Demo.astro)
